### ISSUE TITLE
[Accessibility] [Toggle] Validation fails for button name

### ISSUE THREAD

**Issue by prmittal** [OfficeDev/office-ui-fabric-react#1039]
The toggle component fails accessibility validation for button name. 

Details:
https://dequeuniversity.com/rules/axe/2.1/button-name

Error displayed:

![image](https://cloud.githubusercontent.com/assets/11937034/23118690/09b1d58c-f77b-11e6-99ce-77a51deb78f3.png)

**Comment by battletoilet** [comment]
Needs a text parameter to pass specific text to the aria-label.
